src/cable/connection.cr

```diff
@@ -52,36 +52,43 @@ module Cable
       @connection_rejected = true
     end
 
+    def channels : Array(Channel)
```

Comment:
To ensure there's no ambiguity, should we qualify this `Channel` with the namespace `Cable::Channel` so it's not confused with Crystal's `Channel`?

Reply:
Good call, `Cable::Channel `is used in the codebase and is more explicit.